### PR TITLE
Add new SessionLengthHelper to apps

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
This has #131 merged in.

**Comment by smn** [comment]
When that lands this'll be easier to review.

File: src/utils.js [hunk 1]
```diff
@@ -1015,5 +1015,27 @@ go.utils = {
...
         });
+    },
+
+    attach_session_length_helper: function (im) {
+      // If we have transport metadata then attach the session length
+      // helper to this app
+      if(!im.msg.transport_metadata)
+        return;
+
+      slh = new go.SessionLengthHelper(im, {
```
Comment (justinvdm) [inline src/utils.js]: I think we need a `var` here. Wish jshint was able to check this, but for some reason it doesn't.

**Comment by justinvdm** [comment]
I think we need tests for the `unspecified` and `unknown` cases. Maybe we also need a test for when transport metadata isn't given, but seems unlikely that such a case would happen.

**Comment by justinvdm** [comment]
Oh, actually, we look up `aat_ussd` for the transport metadata, so maybe that case is likely enough to test.

**Comment by justinvdm** [comment]
Looks good, just some minor comments.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**